pkg/statistics/handle/logutil/logutil.go

```diff
@@ -15,11 +15,29 @@
 package logutil
 
 import (
+	"time"
+
 	"github.com/pingcap/tidb/pkg/util/logutil"
 	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 )
 
 // StatsLogger with category "stats" is used to log statistic related messages.
+// Do not use it to log the message that is not related to statistics.
 func StatsLogger() *zap.Logger {
 	return logutil.BgLogger().With(zap.String("category", "stats"))
 }
+
+// StatsSamplerLogger with category "stats" is used to log statistic related messages.
```

Comment:
you can add more comment about the difference about passing the *logger down and calling the logutil.StatsSamplerLogger() multi times.

the latter will create many children loggers out, while the former is just a kind implementation of singleton. We are going to do the sampling for all priority queue log information, so its necessary to use a singleton to collect them all them sampling them.